client-v2/src/shared/components/article/Article.tsx

```diff
@@ -63,9 +63,9 @@ interface ArticleComponentProps {
   onDragOver?: (d: React.DragEvent<HTMLElement>) => void;
   onDragEnter?: (d: React.DragEvent<HTMLElement>) => void;
   onDrop?: (d: React.DragEvent<HTMLElement>) => void;
-  onDelete?: (uuid: string) => void;
+  onDelete?: () => void;
```

Comment:
In a pique of boy scoutery I've made alterations to the deletion signature here as well -- the uuid argument is redundant.